### ISSUE TITLE
Matrix traceback while startup

### ISSUE THREAD

**Issue by palango** [raiden-network/raiden#2042]
Just to have someone look at it as I think we should at least catch this.

```
You are connected to the 'ropsten' network and the DB path is: /Users/paul/.raiden/netid_3/ef4f7c99/log.db
2018-08-02 13:24:43 [debug    ] Matrix homeserver RTT times    [raiden.network.transport.matrix] rtt_times=[('https://transport03.raiden.network', 0.023839), ('https://transport02.raiden.network', 0.125781)]
2018-08-02 13:24:43 [info     ] Automatically selecting matrix homeserver based on RTT [raiden.network.transport.matrix] homeserver=https://transport03.raiden.network rtt=0.023839
2018-08-02 13:24:43 [info     ] Lock 4475859856 acquired on /Users/paul/.raiden/netid_3/ef4f7c99/.lock [filelock]
2018-08-02 13:24:49 [debug    ] starting at block number       [raiden.tasks] current_block=3758370
2018-08-02 13:24:50 [debug    ] STATE CHANGE                   [raiden.raiden_service] node=ef4f7c99 state_change=<Block 3758370>
2018-08-02 13:24:51 [info     ] LOGIN                          [raiden.network.transport.matrix] current_user=@0xef4f7c9962d8baa8e268b72ec6dd4bdf09c84397:transport03.raiden.network homeserver=transport03.raiden.network server_url=https://transport03.raiden.network username=0xef4f7c9962d8baa8e268b72ec6dd4bdf09c84397
2018-08-02 13:24:52 [debug    ] new block                      [raiden.tasks] number=3758371
2018-08-02 13:24:53 [debug    ] STATE CHANGE                   [raiden.raiden_service] node=ef4f7c99 state_change=<Block 3758371>
Traceback (most recent call last):
  File "/Users/paul/Work/venv-raiden/bin/raiden", line 11, in <module>
    load_entry_point('raiden', 'console_scripts', 'raiden')()
  File "/Users/paul/Work/raiden/raiden/__main__.py", line 11, in main
    run(auto_envvar_prefix='RAIDEN')
  File "/Users/paul/Work/venv-raiden/lib/python3.6/site-packages/click/core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "/Users/paul/Work/venv-raiden/lib/python3.6/site-packages/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/Users/paul/Work/venv-raiden/lib/python3.6/site-packages/click/core.py", line 1043, in invoke
    return Command.invoke(self, ctx)
  File "/Users/paul/Work/venv-raiden/lib/python3.6/site-packages/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/Users/paul/Work/venv-raiden/lib/python3.6/site-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/Users/paul/Work/venv-raiden/lib/python3.6/site-packages/click/decorators.py", line 17, in new_func
    return f(get_current_context(), *args, **kwargs)
  File "/Users/paul/Work/raiden/raiden/ui/cli.py", line 1039, in run
    NodeRunner(kwargs, ctx).run()
  File "/Users/paul/Work/raiden/raiden/ui/cli.py", line 893, in run
    app = self._run_app()
  File "/Users/paul/Work/raiden/raiden/ui/cli.py", line 911, in _run_app
    app_ = run_app(**self._options)
  File "/Users/paul/Work/raiden/raiden/ui/cli.py", line 735, in run_app
    raiden_app.start()
  File "/Users/paul/Work/raiden/raiden/app.py", line 131, in start
    self.raiden.start()
  File "/Users/paul/Work/raiden/raiden/raiden_service.py", line 278, in start
    self.start_async().wait()
  File "/Users/paul/Work/raiden/raiden/raiden_service.py", line 258, in start_async
    self.transport.start(self, queueids_to_queues)
  File "/Users/paul/Work/raiden/raiden/network/transport/matrix.py", line 196, in start
    self._join_discovery_room()
  File "/Users/paul/Work/raiden/raiden/network/transport/matrix.py", line 356, in _join_discovery_room
    discovery_room = self._client.join_room(self._discovery_room_alias_full)
  File "/Users/paul/Work/venv-raiden/lib/python3.6/site-packages/matrix_client/client.py", line 320, in join_room
    response = self.api.join_room(room_id_or_alias)
  File "/Users/paul/Work/venv-raiden/lib/python3.6/site-packages/matrix_client/api.py", line 195, in join_room
    return self._send("POST", path)
  File "/Users/paul/Work/venv-raiden/lib/python3.6/site-packages/raiden_libs/network/matrix/client.py", line 48, in _send
    return super()._send(*args, **kwargs)
  File "/Users/paul/Work/venv-raiden/lib/python3.6/site-packages/matrix_client/api.py", line 691, in _send
    code=response.status_code, content=response.text
matrix_client.errors.MatrixRequestError: 500: {"errcode":"M_UNKNOWN","error":"Internal server error"}
```

**Comment by andrevmatos** [comment]
@ulope any server logs which my hint on what happened here? transport03

**Comment by andrevmatos** [comment]
Shoud be fixed by #2062
